Adds note re: local users (#979)
sensu/sensu-docs: content/sensu-enterprise-dashboard/2.12/rbac/rbac-for-ldap.md

}
{{< /highlight >}}

> _NOTE: You'll notice in the example there are no attributes defined in dashboard.users. This is due to the way that the LDAP integration works. When using this provider, there will not be an option to have local users as a fall back option._

Reviewer: ```suggestion
_NOTE: You'll notice in the example there are no attributes defined in dashboard.users. When using this provider, there will not be an option to have local users as a fallback option._
```
